README.md

```diff
@@ -68,6 +68,15 @@ If needed you can pass `{ exposeStatusRoute: true }` and `under-pressure` will e
 
 If you need the change the exposed route path, you can pass `{ exposeStatusRoute: '/alive' }` options.
 
+The options object will be passed down to the route registration method to allow passing of route configuration options.
+```js
+fastify.register(require('under-pressure'), {
+  maxEventLoopDelay: 1000,
+  exposeStatusRoute: true,
```

Comment:
I would prefer to expose the route options as an object inside `exposeStatusRoute: routeOpts` or similar. There might be options in the future that could clash with the ones in fastify, and sending them all down might become problematic.

Reply:
Good idea. I've updated the PR.